### ISSUE TITLE
Spurious points in radial chart

### ISSUE THREAD

**Issue by FabienDaniel** [plotly/plotly.js#2023]
Hi,

I try to set up a radar chart and a few spurious points appear. I was not sure, if this was a bug or a bad use and posted that issue on [stackoverflow](https://stackoverflow.com/questions/46295498/radar-chart-with-plotly-python-3x)
```

import plotly.offline as pyo
pyo.init_notebook_mode()
from plotly.graph_objs import *
import plotly.graph_objs as go

tr1 = (go.Scatter(r      = [6, 7],
                  t      = [120, 125],
                  mode   = 'markers',
                  name   = 'Drama',
                  text   = ['text_1', 'text_2'],
                  marker = dict(color = 'royalblue', size=110,
                                       line=dict(color='white'), opacity=0.7),
                  hoverinfo = 'all'
                  )) 
layout = go.Layout(
    title='Test Case',
    font=dict(
        size=15
    ),
    plot_bgcolor='rgb(223, 223, 223)',
    angularaxis=dict(        
        tickcolor='rgb(253,253,253)'
    ),
    hovermode='Closest',
)
fig = go.Figure(data = [tr1], layout=layout)
pyo.iplot(fig)
```

My main issue is that two points appear at coordinates: (t,r) = (3,12) and (4,13). Also, I want that 'text_1' and 'text_2' appear on hover but currently, only the points coordinates are shown.

The plot is visible on this [Kaggle](https://www.kaggle.com/fabiendaniel/categorizing-actors-hands-on-plotly) page.

Thanks for the help !

**Comment by etpinard** [comment]
Looks like a bug.

**Comment by etpinard** [comment]
fixed in `scatterpolar` after #2200
